protos/flyteidl/admin/launch_plan.proto

```diff
@@ -91,6 +92,9 @@ message LaunchPlanSpec {
     Auth auth                             = 8 [deprecated=true];
 
     AuthRole auth_role                    = 9;
+
+    // Indicates the amount of queueing a launched workflow execution can tolerate.
+    QualityOfService quality_of_service   = 16;
```

Comment:
Since this will be part of the launchplan should we also remove [this](https://github.com/lyft/flyteidl/blob/e279a7eb92b56f726fe13795e4f088976b14f752/protos/flyteidl/core/workflow.proto#L132)?

Reply:
I believe we'll want to resolve the queueing budget based on the quality of service on the admin side before creating the workflow CRD so we should still keep the queueing_budget field

Reply:
thank you for the catch! updated to use QualityOfService in the workflow metadata